stable/app/templates/cron.yaml

```diff
@@ -1,12 +1,15 @@
 {{- if .Values.cron.enabled }}
+{{- $fullName := include "app.fullname" . -}}
+{{- $appLabels := include "app.labels" . -}}
+{{- $appSelectorLabels := include "app.selectorLabels" . -}}
 {{- range $job := .Values.cron.jobs }}
 apiVersion: batch/v1
 kind: CronJob
 metadata:
-  name: "{{ include "app.fullname" . }}-{{ $job.name }}"
-  namespace: {{ .Release.Namespace }}
+  name: {{ printf "%s-%s" $.Release.Namespace $job.name | trunc 63 | trimSuffix "-" }}
```

Comment:
Whats the reason of using `Namespace` as a prefix here?
Also truncating the name can still lead to collisions in name. Shall we add a checksum as a suffix to stay in character limit and still have no collisions.

Reply:
1. The chart was earlier using `app.fullName` as a prefix but that became too long sometimes, and out of the other identifying names - I used `.Release.Namespace` as its value can be considerably smaller while still grouping the jobs with a common prefix.

2. Added slice index numbers to make them unique. 
`dry-run` output:
```text
cronjob.batch/guardian-0-fetch-resources created (dry run)
cronjob.batch/guardian-1-appeal-expiration-reminder created (dry run)
cronjob.batch/guardian-2-appeal-expiration-revocation created (dry run)
```

Reply:
I feel namespace is too generic and not very useful, can we keep the `app.fullname` with some truncation logic itself ?
Like let's keep 30 chars for full name and 30 chars for job name rest 3 characters for the index numbers.